packages/twenty-front/src/modules/workflow/workflow-trigger/utils/getTriggerIcon.ts

```diff
@@ -6,6 +6,13 @@ import { OTHER_TRIGGER_TYPES } from '@/workflow/workflow-trigger/constants/Other
 export const getTriggerIcon = (
   trigger: WorkflowTrigger,
 ): string | undefined => {
+  if (
+    trigger.settings &&
```

Comment:
@thomtrp @martmull does it make sense to you to use the trigger.icon here?

I would say yes.

Code wise: please use isNonEmtyString(this.trigger?.settings?.icon)